prow/cmd/deck/main.go

```diff
@@ -77,6 +78,7 @@ func (o *options) Validate() error {
 func gatherOptions() options {
 	o := options{}
 	flag.StringVar(&o.configPath, "config-path", "/etc/config/config.yaml", "Path to config.yaml.")
+	flag.StringVar(&o.jobConfigPath, "job-config-path", "", "Path to prow job configs.")
```

Comment:
Should we provide a default for this flag like we do for `config-path`? We aren't consistent about this across binaries (`config-path` doesn't always have a default specified).

Reply:
if we do it now it will break other people's prow deployments :-)

Reply:
Ah, good point. I forgot that we are still allowing this to be unspecified.